### PR TITLE
Ghost ERT message formatting

### PR DESCRIPTION
**Pull request by Shad0vvs**
## Why It's Good For The Game

Formatting:

After and before:
Untitled

## Changelog

:cl:
fix: Fixed color formatting on ghost ERT alerts.
/:cl:

### CODE REVIEW

**Review by stanalbatross — APPROVED** [review]

File: code/datums/emergency_calls/emergency_call.dm [hunk 1]
```diff
@@ -91,7 +91,7 @@
 
 	for(var/mob/dead/observer/M in GLOB.observer_list)
 		if(M.client)
-			to_chat(M, FONT_SIZE_LARGE("\n<span class='attack'>[ert_message]. >>> <a href='?src=\ref[M];joinresponseteam=1;'><b>Join Response Team</b></a></span> <<<"))
+			to_chat(M, FONT_SIZE_LARGE("\n<span class='attack'>[ert_message]. >>> <a href='?src=\ref[M];joinresponseteam=1;'><b>Join Response Team</b></a> <<<</span>"))
```
Comment (Watermelon914) [inline code/datums/emergency_calls/emergency_call.dm:94]: you should not be putting in >>> and <<< in html statements, these should be &lt; and &gt;

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

File: code/datums/emergency_calls/emergency_call.dm (hunk 1 above)
Comment (Watermelon914) [reply code/datums/emergency_calls/emergency_call.dm]: sorry this was a mistake because html memes, you need to use `&lt;` and `&gt;`

**Review by stanalbatross — CHANGES_REQUESTED** [review]
do walter's change

**Review by stanalbatross — APPROVED** [review]
even though it's in italics

**Pull request merged**